python/paddle/fluid/reader.py

```diff
@@ -298,6 +381,36 @@ def decorate_sample_generator(self,
                 is less than batch_size. 
             places (None|list(CUDAPlace)|list(CPUPlace)): place list. Must
                 be provided when PyReader is iterable.
+
+        Example:
+            .. code-block:: python
+
+                import paddle.fluid as fluid
+                import numpy as np
+
+                EPOCH_NUM = 3
+                ITER_NUM = 5
+
+                def random_image_generator(height, width):
+                    def generator():
+                        for i in range(ITER_NUM):
+                            yield np.random.uniform(low=0, high=255, size=[1, height, width])
+                    return generator
+
+                image = fluid.layers.data(name='image', shape=[784, 784], dtype='float32')
+                reader = fluid.io.PyReader(feed_list=[image], capacity=4, iterable=True)
+
+                user_defined_generator = random_image_generator(784, 784)
+                reader.decorate_sample_generator(user_defined_generator,
+                                                 batch_size=8,
+                                                 places=[fluid.core.CUDAPlace(0)])
```

Comment:
`fluid.core.CUDAPlace(0)` -> `fluid.CUDAPlace(0)`.

Reply:
When using `decorate_sample_generator`, reader should yield a list/tuple of sample data. Please fix it.

Reply:
Done